### PR TITLE
Update callouts about data retention

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

There was another change in the data retention policy for retrieving orders from more than 2 years ago. Now, it is no longer possible to open a ticket for support for data requests, because VTEX doesn't store that information anymore. In those endpoints, there was the same callout about this, and I have removed a sentence that said it was possible to open a ticket.

### CODE REVIEW

File: VTEX - Orders API.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
...
...Ean\": null,\r\n                \"ImageUrl\": \"http://ppxps.vteximg.com.br/arquivos/ids/155407-55-55/EE803C74-37A5-4804-B1A6-9F12D22EA505.png?v=637559269453730000\",\r\n                \"DetailUrl\": \"/camiseta/p\",\r\n                \"AssemblyOptions\": []\r\n            }\r\n        ]\r\n    },\r\n    \"subscriptionData\": {\r\n        \"SubscriptionGroupId\": \"C191822AF072C7508F9BBBC655FE8E60\",\r\n        \"Subscriptions\": [\r\n            {\r\n                \"ExecutionCount\": 274,\r\n                \"PriceAtSubscriptionDate\": 2265.0,\r\n                \"ItemIndex\": 0,\r\n                \"Plan\": {\r\n                    \"type\": \"RECURRING_PAYMENT\",\r\n                    \"frequency\": {\r\n                        \"periodicity\": \"DAILY\",\r\n                        \"interval\": 1\r\n                    },\r\n                    \"validity\": {\r\n                        \"begin\": \"2021-10-05T00:00:00.0000000+00:00\",\r\n                        \"end\": null\r\n                    }\r\n                }\r\n            }\r\n        ]\r\n    },\r\n    \"taxData\": null,\r\n    \"checkedInPickupPointId\": null,\r\n    \"cancellationData\": null\r\n}\r\n```",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:33]: Remover o response body, melhor deixar ele só no schema, e adicionar as permissões na `description`

File: VTEX - Orders API.json [hunk 2]
```diff
@@ -1644,7 +1644,7 @@
...
... return the `items` property. \r\n\r\n> You can only access information from orders created in the last two years, and that same period is valid for customers through [My Account](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh). \r\n\r\n> This endpoint should **not** be used for integrations. Use the [orders Feed or hook](https://developers.vtex.com/docs/guides/orders-feed) for this purpose. \r\n\r\nThis endpoint returns only orders that already have been indexed, which takes approximately four minutes. Because of this, the data retrieved may present inconsistencies. To get live up-to-date information and [build order integrations](https://developers.vtex.com/vtex-rest-api/docs/erp-integration-set-up-order-integration) use the [orders Feed or hook](https://developers.vtex.com/docs/guides/orders-feed).\r\n\r\n> **Onboarding guide** \r\n>\r\n> Check the new [Orders onboarding guide](https://developers.vtex.com/vtex-rest-api/docs/orders-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Orders and is organized by focusing on the developer's journey.",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:1647]: Adicionar premissões

File: VTEX - Orders API.json [hunk 3]
```diff
@@ -4404,7 +4404,7 @@
...
+                "description": "Lists all orders from a given customer, filtering by their email. \r\n\r\n> You can only access information from orders created in the last two years, and that same period is valid for customers through [My Account](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh). \r\n\r\n> Note that this request should be made by an [user](https://developers.vtex.com/docs/guides/user-authentication-and-login) or [an appKey / appToken pair](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) that is associated with the [Call center operator](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy#call-center-operator) role. Otherwise, it will return only orders from the same email informed in the `clientEmail` query parameter.",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:4407]: Adicionar permissões

File: VTEX - Orders API.json [hunk 4]
```diff
@@ -4920,7 +4920,7 @@
...
+                "description": "Lists all details from an order, through the perspective of the customer who placed the order. \r\n\r\n> You can only access information from orders created in the last two years, and that same period is valid for customers through [My Account](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh). \r\n\r\n> Note that this request should be made by an [user](https://developers.vtex.com/docs/guides/user-authentication-and-login) or [an appKey / appToken pair](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) that is associated with the [Call center operator](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy#call-center-operator) role. Otherwise, it will return only orders from the same email informed in the `clientEmail` query parameter.",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:4923]: Adicionar permissões

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Orders API.json (hunk 1 above)
Comment (Isabella-Veloso) [reply VTEX - Orders API.json:33]: Ok, response body removido e permissões adicionadas.

File: VTEX - Orders API.json (hunk 3 above)
Comment (Isabella-Veloso) [reply VTEX - Orders API.json:4407]: Ok, permissões adicionadas.

File: VTEX - Orders API.json (hunk 4 above)
Comment (Isabella-Veloso) [reply VTEX - Orders API.json:4923]: Ok, permissões adicionadas.

File: VTEX - Orders API.json (hunk 2 above)
Comment (Isabella-Veloso) [reply VTEX - Orders API.json:1647]: Ok, permissões adicionadas.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**